packages/gatsby-transformer-screenshot/src/gatsby-node.js

```diff
@@ -1,8 +1,19 @@
 const crypto = require(`crypto`)
 const axios = require(`axios`)
+const Queue = require(`better-queue`)
 const { createRemoteFileNode } = require(`gatsby-source-filesystem`)
 
 const SCREENSHOT_ENDPOINT = `https://h7iqvn4842.execute-api.us-east-2.amazonaws.com/prod/screenshot`
+const LAMBDA_CONCURRENCY_LIMIT = 50
```

Comment:
Should this be configurable via environment variable, perhaps?

Reply:
There's only one lambda that we own so unless we make the endpoint configurable, there's no reason not to hard code this.